### ISSUE TITLE
Keep one of a pair option

### ISSUE THREAD

**Issue by msdobrescu** [pixelb/fslint#166]
Hello, assuming the comparison finds duplicated in one folder, could be selected all the duplicates from a directory, but, usually, one should be kept, so not selected. Add this option, when selecting all the files form a directory, like "Select all in this directory but first", as in the groups case, because can't work with groups selection in all the cases.

**Comment by endolith** [comment]
Yes, this is the biggest problem I have with fslint.  I try to delete a handful of duplicates, selecting only the ones in a particular folder:

![2020-04-23 20_52_01-shuttle (omegatron@shuttle) - VNC Viewer](https://user-images.githubusercontent.com/58611/80163562-89211980-85a4-11ea-8028-105b9f86834d.png)

but then it chokes every time there's a group with all the files from the same folder:


![2020-04-23 20_55_05-shuttle (omegatron@shuttle) - VNC Viewer](https://user-images.githubusercontent.com/58611/80163732-13697d80-85a5-11ea-86af-aa78f7c75a29.png)

and I have to say No, then very carefully hold down Ctrl and manually unselect all of the files in the group, and then press Delete again, and then it pops up another warning, etc. etc. etc. until I finally get to the end and it actually deletes them all.

I'm glad it asks me if I want to delete all of the ones in the group before just deleting them, but it shouldn't just completely give up when I say no.  AllDup is much better about this, you can just tell it to skip those and then deal with them later when the rest have been taken care of:

![2020-04-23 20_56_51-File Manager - Process selected files_ 1 (54 02 KB)](https://user-images.githubusercontent.com/58611/80163739-1c5a4f00-85a5-11ea-983b-654493f6954b.png)


but I don't trust AllDup  over network shares / Linux filesystems because I'm afraid it will not handle symlinks or metadata correctly, etc.
